modules/installing-ocp-agent-ibm-z-zvm.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_with_agent_based_installer/prepare-pxe-infra-agent.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-ocp-agent-ibm-z-zvm_{context}"]
+= Adding {ibm-z-title} agents with z/VM
+
+Use the following procedure to manually add {ibm-z-name} agents with z/VM
```

Comment:
```suggestion
Use the following procedure to manually add {ibm-z-name} agents with z/VM.
```

Reply:
Covered in this PR: https://github.com/openshift/openshift-docs/pull/73204